sources/packages/web/src/components/common/ProgramOfferingDetailHeader.vue

```diff
@@ -12,8 +12,11 @@
       />
       <v-divider-vertical-opaque class="mx-2 my-0" />
       <header-title-value title="Institution name"
-        ><template #value
-          ><span class="link-primary" @click="goToInstitutionProfile()">
+        ><template #value>
+          <span v-if="isReadOnlyUser($props.locationId)">
+            {{ headerDetails.institutionName }}
+          </span>
+          <span v-else class="link-primary" @click="goToInstitutionProfile()">
```

Comment:
Executing the check based on the `isReadOnlyUser` would not be accurate.
If a permission needed to be checked for this it must be based on the user being an admin or not, but it would be outside the scope of this PR.